apps/e2e/ag-grid-storybook/src/app/shared/baseball-players-data.ts

```diff
@@ -62,6 +62,7 @@ export const columnDefinitions: ColDef[] = [
     headerName,
     sortable: false,
     cellClass: 'booleanType',
+    cellRenderer: 'textContent',
```

Comment:
is this because it renders a checkbox for boolean values now? Is this something we might need to add to our default colDef or text column type in `getGridOptions`?

Reply:
Yeah, I have a note to add it to the migration guide. I'm reluctant to add additional grid options logic around it.

Booleans are now rendered with their new checkbox renderer. There might be a story around making the checkbox use our checkbox. I'll add that topic for tomorrow.